Specs/Core/HeadingPitchRollSpec.js

```diff
@@ -0,0 +1,91 @@
+/*global defineSuite*/
+defineSuite([
```

Comment:
Do these tests cover all the new `HeadingPitchRoll` functions?  We would want 100% test coverage for that file.  If you want to coverage, see https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide#run-all-tests-with-code-coverage-build-instrumentforcoverage-first